aws/awss3/options/s3presigned/s3_presigned.go

```diff
@@ -10,9 +10,21 @@ type OptionS3Presigned interface {
 
 type confS3Presigned struct {
 	PresignFileName string
+	PresignFileType *ContentDispositionType // inline or attachment
 	PresignExpires  time.Duration
 }
 
+type ContentDispositionType string
+
+const (
+	PresignFileTypeInline     ContentDispositionType = "inline"
+	PresignFileTypeAttachment ContentDispositionType = "attachment"
+)
+
+func (o *ContentDispositionType) String() string {
+	return string(*o)
+}
```

Comment:
分岐を減らすために attachment を defaultにするのが良いと思います。

```suggestion
type confS3Presigned struct {
	PresignFileName string
	ContentDispositionType ContentDispositionType
	PresignExpires  time.Duration
}

type ContentDispositionType int

const (
	ContentDispositionTypeAttachment ContentDispositionType = iota
	ContentDispositionTypeInline
)
```

Reply:
https://github.com/88labs/go-utils/pull/77/commits/e5630f33f0eae249b7c3d82acaff3cb146b631a9